include/quda.h

```diff
@@ -550,6 +554,8 @@ extern "C" {
     int batched_rotate;
     /** For block method solvers, the block size **/
     int block_size;
+    /** The block size when computing eigenvalues **/
```

Comment:
I'd make this a bit clearer since on first (and second, and third) read it's hard to grok why this is different from `block_size`. Maybe call the variable `eval_compute_batch_size` or something like that? Clarifying that (a) it's for the eigenvalue computation and (b) that it's just batching work, not a formal block algorithm.

Reply:
fc0762d6152ff9b3f61b4c5d1787273c3d64ca02